File: ruby/examples/anthropic_messages.rb
# frozen_string_literal: true

require 'buble'

client = Buble::Client.new

message = client.chat.messages.create(
  model: 'openai/gpt-5.4',
  system: 'You are concise.',
  messages: [
    { role: 'user', content: 'Summarize this release.' }
  ],
  max_tokens: 800
)

puts message

File: php/examples/anthropic-messages.php
<?php

declare(strict_types=1);

require __DIR__ . '/../vendor/autoload.php';

use Buble\BubleClient;

$client = BubleClient::fromEnv();

$message = $client->chat()->messages()->create([
    'model' => 'openai/gpt-5.5',
    'system' => 'You are concise.',
    'messages' => [
        ['role' => 'user', 'content' => 'Summarize this release.'],
    ],
    'max_tokens' => 800,
]);

echo $message['content'][0]['text'] . PHP_EOL;
